addons/mail/static/src/public/discuss_public_boot.js

```diff
@@ -54,24 +52,25 @@ Component.env = legacyEnv;
         startServices(env),
         odoo.loadTemplatesPromise.then(processTemplates),
     ]);
-    env.qweb.addTemplates(templates);
     mapLegacyEnvToWowlEnv(Component.env, env);
     odoo.isReady = true;
     legacyServiceRegistry.add('messaging', MessagingService.extend({
         messagingValues: {
             autofetchPartnerImStatus: false,
         },
     }));
-    await mount(MainComponentsContainer, { env, target: document.body });
+    await mount(MainComponentsContainer, document.body, { env, templates });
     createAndMountDiscussPublicView();
```

Comment:
Maybe pass `templates` as param to avoid having to fetch it from `legacySession.owlTemplates` twice below?
Also probably should pass `env` as param, it doesn't matter for now as we get the global legacy one but once we use the new one we will want the one coming from `makeEnv` above too.